### PR TITLE
Fix invoice number in multishop context

### LINKED ISSUE #18749: Invoice prefix not working in multishop context
#### Describe the bug

Invoice prefix not working in multishop context

#### Expected behavior

Invoice prefix should working in multishop context

#### Steps to Reproduce

0. Setup a Prestashop instance with multishop enabled
1. Go to Orders > Invoices
2. Set a custom invoice prefix
3. Download an invoice from an order
4. Invoice prefix is not displayed on top right of the invoice

#### Additional information

* PrestaShop version: 1.7.6.5
* Fixed in PR: #18696

### PR DESCRIPTION
**Pull request by simondaigre**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix invoice number in multishop context
| Type?         | bug fix
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | #18749 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18696)

### CODE REVIEW

File: classes/pdf/HTMLTemplateInvoice.php [hunk 1]
```diff
@@ -71,7 +71,9 @@ public function __construct(OrderInvoice $order_invoice, Smarty $smarty, $bulk_m
         $this->date = Tools::displayDate($order_invoice->date_add);
 
         $id_lang = Context::getContext()->language->id;
-        $this->title = $order_invoice->getInvoiceNumberFormatted($id_lang);
+        $id_shop = Context::getContext()->shop->id;
+
```
Comment (Progi1984) [inline classes/pdf/HTMLTemplateInvoice.php]:
```suggestion

```

**Review by Progi1984 — COMMENTED** [review]
Please fix PHPCS errors. Thanks :)

**Comment by simondaigre** [comment]
@Progi1984 done !

**Review by PierreRambaud — APPROVED** [review]

**Review by matthieu-rolland — DISMISSED** [review]

**Comment by matthieu-rolland** [comment]
@simondaigre thank you for your PR !

Could you please create an issue explaining the bug and how to reproduce it ? So that our QA team can test your modification? 

https://github.com/PrestaShop/PrestaShop/issues/new?template=bug_report.md

(then you can put the link of the issue in the description of this PR)

Thanks again 👍

**Review by matthieu-rolland — DISMISSED** [review]
@simondaigre thank you for your PR !

Could you please create an issue explaining the bug and how to reproduce it ? So that our QA team can test your modification?

https://github.com/PrestaShop/PrestaShop/issues/new?template=bug_report.md

(then you can put the link of the issue in the description of this PR)

Thanks again 👍

**Review by Progi1984 — APPROVED** [review]

**Comment by simondaigre** [comment]
See issue #18749

**Comment by khouloudbelguith** [comment]
Hi @simondaigre,

It is ok :heavy_check_mark: 

Shop1
image
Shop2
image

Thanks!

**Comment by Progi1984** [comment]
Thanks @simondaigre

**Pull request merged**